Payments/Integration/01_ManageConnection.md

```diff
@@ -0,0 +1,140 @@
+
+If you want to establish a new connection to a payment service provider (PSP), you can buy, depending on the payment service provider, the needed connection in the app store or license it directly at the payment service provider. By default, Actindo supports the following drivers:
+- Adyen&trade;
+- Amazon pay&trade;
+- EBICS&trade;
+- Stripe&trade;
+- A dummy driver to simulate payments for testing purposes    
+
+The data structure of the connection then provides exactly the data that the PSP needs to exchange data with the Actindo *Payments* module.
+
+In the following, the general settings to create, edit, 
+or disable connections are described. 
+
+
+## Create a PSP connection
+
+Create a connection to a certain payment provider. As soon you have established a connection, an initial synchronization will be triggered, which will transfer the entire data model of the connected payment service provider to Actindo. 
+
+
+#### Prerequisites
+
+At least one connection has been installed.
+
+
+#### Procedure    
+
+*Payments > Settings > Tab CONNECTIONS*   
+
+![Connections](../../Assets/Screenshots/Payments/Settings/Connections.png "[Connections]")
+1.  Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+    The view for creating a connection is displayed.   
+    ![Create connection](../../Assets/Screenshots/Payments/Settings/CreateConnection.png "[Create connection]")
+2.  Enter a name for the connection in the *Name* field.
+3.  Click the *Driver* drop-down list and select the desired driver. 
+   All already installed drivers are displayed in the list.   
+   
+     > [Info] If you want to install additional drivers, select the **more...** option and buy a driver in the app store. You have to purchase drivers from the main account. You need to install them separately in each sandbox or account where you want to use it.
+
+     The *Credentials* section is displayed below the drop-down list.
+
+    ![Credentials](../../Assets/Screenshots/Payments/Settings/CreateConnectionCredentials.png "[Credentials]")
+     > [Info] Depending on the selected driver, the fields in the *Credentials* section differ. 
+
+4. Enter the required data in the fields in the *Credentials* section. Use the information you received from the payment service provider to fill the credentials.   
+5. If the PSP connection requires additional settings, open the *Settings* tab and add further settings.
+
+6. Click the [SAVE] button.   
+    The connection will be established. The *Checking credentials...* notice is displayed. After a few seconds, the *Loading data...* notice is displayed.
+
+     ![Checking credentials](../../Assets/Screenshots/Payments/Settings/CheckingCredentials.png "[Checking credentials]")
+
+    The view for creating connections is automatically closed when the connection has been established. The *Edit connection* view of the newly created connection is displayed. The *Settings* tab is selected.
```

Comment:
Im Sandbox (Example driver) wird die (List of) Connections wieder angezeigt, nicht die Settings tab der erstellten Verbindung, aber am Besten Stefan fragen.